packages/builder/src/components/common/LinkedRecordSelector.svelte

```diff
@@ -0,0 +1,74 @@
+<script>
+  import { onMount } from "svelte"
+  import { backendUiStore } from "builderStore"
+  import api from "builderStore/api"
+
+  export let modelId
+  export let linkName
+  export let linked = []
+
+  let records = []
+
+  async function fetchRecords() {
+    const FETCH_RECORDS_URL = `/api/${$backendUiStore.selectedDatabase._id}/${modelId}/records`
+    const response = await api.get(FETCH_RECORDS_URL)
+    records = await response.json()
+  }
+
+  onMount(() => {
+    fetchRecords()
+  })
+
+  function linkRecord(record) {
+    linked.push(record._id)
+  }
+</script>
+
+<section>
+  <h3>{linkName}</h3>
+  {#each records as record}
+    <div class="linked-record" on:click={() => linkRecord(record)}>
+      <div class="fields">
+        {#each Object.keys(record).slice(0, 2) as key}
```

Comment:
exclude internal columns, probably... `_id` , `modelId`, `type`, `_rev`

Reply:
Yep this is updated on my branch 👍